socket/server.js

```diff
@@ -0,0 +1,57 @@
+"use strict"; 
+const express = require('express'); 
+const http = require('http');
+const socketIo = require("socket.io");
+const { PubSub } = require('@google-cloud/pubsub');
+
+// Setting up express and adding socketIo middleware
+const app = express();
+const server = http.createServer(app);
+const io = socketIo(server);
+
+const port = process.env.PORT || 8082;
+
+io.on("connection", (socket) => {
+  console.log("New client connected");
```

Comment:
dump the port here so you can distinguish between different instances once you scale this